### ISSUE TITLE
HDM container should be able to make use of a different port.

### ISSUE THREAD

**Issue by tuxmea** [betadots/hdm#205]
Even when using hdm Puppet module we can not set a different port, as the container seems to have port 3000 hardcoded.

**Comment by rwaffen** [comment]
yes the EXPOSE of port 3000 for the container is hardcoded. we have to change this then.

**Comment by oneiros** [comment]
FWIW, our default puma config in `config/puma.rb` allowes overriding the default port by setting the environment variable `PORT`.

**Comment by tuxmea** [comment]
fixed with #257
